downstairs/src/lib.rs

```diff
@@ -3333,12 +3333,6 @@ impl Work {
                 // return idempotently.
                 true
             }
-            WorkState::Done => {
```

Comment:
We can remove the comment up on 3235 as it's not possible for that situation
to exist.

```
If this job is not new, then just return `false`.
```